api/api.js

```diff
@@ -4,6 +4,7 @@ const mongodb = require('mongodb')
 const cors = require('cors')
 const { check, validationResult } = require('express-validator')
 const fetch = require('node-fetch')
+const { Long } = require('bson')
```

Comment:
https://www.npmjs.com/package/long

is much smaller and efficient

Reply:
Thanks! I just used the dependency that came with MongoDB